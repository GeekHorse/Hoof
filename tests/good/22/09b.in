new right bar foo done
new down bar done
new down baz done
most up
dig bar done
value
save
