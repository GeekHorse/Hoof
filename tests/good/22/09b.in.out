new right bar foo done
new down bar done
new down baz done
